### ISSUE TITLE
prompt dialog - feature request

### ISSUE THREAD

**Issue by wizlif** [Codigami/CFAlertDialog#11]
### New Issue Checklist

* [ x] I have read and understood the [CONTRIBUTING guide](https://github.com/Codigami/CFAlertDialog/blob/develop/.github/CONTRIBUTING.md)
* [ x] I have read the [Documentation](https://github.com/Codigami/CFAlertDialog/blob/develop/README.md)
* [ x] I have searched for a similar issue in the [project](https://github.com/Codigami/CFAlertDialog/issues) and found none

### Issue Info

 Info                          | Value                               |
-------------------------------|-------------------------------------|
 Platform Name                 | e.g. Android
 Platform Version              | e.g. 6.0
 CFAlertDialog Version         | e.g. 2.1.1
 Integration Method            | e.g. gradle / manually
 Xcode Version                 | e.g. Android Studio 2.3
 Repro rate                    | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj       | e.g. does it happen with our demo project?
 Demo project link             | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

This is just a feature request, for an edit text component with a callback response for the text entered. Like prompt dialogs
